src/docs/sphinx/dev_guide/mfem_dev.rst

```diff
@@ -3,58 +3,27 @@
 .. ## SPDX-License-Identifier: (BSD-3-Clause)
 
-======================
-MFEM Development Build
-======================
+===========================
+MFEM/Axom Development Build
+===========================
 
-Streamlining back-porting features to MFEM
-------------------------------------------
+Co-Developing with Axom and MFEM
+--------------------------------
 
-Occasionally, Serac developers may want to back-port useful features back to MFEM. MFEM has two separate build systems (Makefile and CMake).
-The default Makefile system that is suggested requires installing suggested packages. Fortunately, MFEM's CMake build system is more compatible with Serac's CMake-based build system.
-Since Serac provides tools to build MFEM's dependencies, one can streamline the back-porting process by re-using Serac's existing host-config files located in `<SERAC_ROOT_DIR>/host-configs`.
+Occasionally, Serac developers may want to patch fixes and useful features back to MFEM and Axom. A co-development build
+environment is available to expedite this process via `git submodules <https://git-scm.com/book/en/v2/Git-Tools-Submodules>`_.
 
-MFEM's cmake build instructions suggest copying a `default.cmake` file to `user.cmake` and making adjustments to CMake variables. If we use Serac's host-config files, we need to only make a few modifications to `user.cmake`, and we can avoid rebuilding MFEM's dependencies manually.
-
-To copy over MFEM's `default.cmake` file:
+To enable a co-develop workflow, use the following commands from the root serac directory:
 
 .. code-block:: bash
 
-    $ cd <mfem-root>
-    $ cp config/defaults.cmake config/user.cmake
-
-
-Assuming the default serac build configuration we need to make the following modifications:
-
-.. code-block:: cmake
-
-    # change MFEM_USE_MPI from OFF to ON
-    option(MFEM_USE_MPI "Enable MPI parallel build" ON)
-    #...
-    # Serac's default build also includes
-    option(MFEM_USE_SUNDIALS "Enable SUNDIALS usage" ON)
-    option(MFEM_USE_SUPERLU "Enable SuperLU_DIST usage" ON)
-    option(MFEM_USE_NETCDF "Enable NETCDF usage" ON)    
-    #...
-    # For use with CUDA, turn the following ON
-    option(MFEM_USE_AMGX "Enable AmgX usage" OFF)
-    option(MFEM_USE_CUDA "Enable CUDA" OFF)
-    #...
-    # uncomment HYPRE_REQUIRED_PACKAGES
-    set(HYPRE_REQUIRED_PACKAGES "BLAS" "LAPACK" CACHE STRING
-      "Packages that HYPRE depends on.")
-
-
-Afterwards create a MFEM build directory and run ``cmake`` using a Serac host-config.
-
-.. code-block:: bash
-		
-    $ mkdir <build>
-    $ cd <build>
-    $ cmake -C <serac/host-config.cmake> ..
+   # Initialize the submodules for MFEM and Axom.
+   $ git submodule update --init --recursive
 
+   # Configure our CMake build with the co-develop option using a local host-config (CMake configuration file) generated by spack.
+   # This uses CMake to configure Axom, MFEM, and Serac in one step.
+   $ ./config-build -hc <host config file> -DSERAC_ENABLE_CODEVELOP=On
```

Comment:
I understand that this is probably the preferred way of building on LC machines, but can't we also just configure cmake normally? e.g. I build serac without ever invoking `config-build`, with

```
/path/to/serac $ cmake . -C path/to/spack/generated/cache.cmake -B build -DSERAC_ENABLE_CODEVELOP=On
```

I imagine using `cmake` directly is probably more intuitive for people outside of LLNL

Reply:
I tentatively want to say yes, as I think I was doing this with the vscode plugin. Any thoughts @white238 ?

Reply:
I am fine with providing both.  I'll do a small PR shortly.

Side Note @samuelpmishLLNL : always put the path to the directory your `CMakeLists.txt` at the end of your cmake command.  Unfortunately CMake's command line arguments are order dependent and I have found very very hard to figure out behavior differences between putting it at the front or end of the command line.

```
mkdir build && cd build
cmake -C path/to/initialcache.cmake  ..
```